tests/test-core/test-object.js

```diff
@@ -0,0 +1,130 @@
+/* globals describe, before, it */
+
+'use strict'
+
+const expect = require('chai').expect
+const IPFS = require('../../src/ipfs-core')
+const bs58 = require('bs58')
+const mDAG = require('ipfs-merkle-dag')
+const DAGNode = mDAG.DAGNode
+const DAGLink = mDAG.DAGLink
+
+// TODO use arrow funtions again when https://github.com/webpack/webpack/issues/1944 is fixed
+describe('object', function () {
+  var ipfs
+
+  before(function (done) {
+    ipfs = new IPFS()
+    done()
+  })
+
+  it('new', function (done) {
+    ipfs.object.new(function (err, obj) {
+      expect(err).to.not.exist
+      expect(obj).to.have.property('Size')
+      expect(obj.Size).to.equal(0)
+      expect(obj).to.have.property('Name')
+      expect(obj.Name).to.equal('')
+      expect(obj).to.have.property('Hash')
```

Comment:
This could be nicer like this:

```js
expect(obj).to.have.property('Size')
  .that.is.equal(0)

expect(obj)
  .to.have.property('Name')
  .that.is.equal('')
```